docs/mint.json

```diff
@@ -90,7 +90,6 @@
     {
       "group": "Tools",
       "pages": [
```

Comment:
⚠️ Removing this page will break several existing documentation links. This page is referenced in:
1. docs/introduction/intro/basic.mdx
2. docs/introduction/intro/quickstart_3.mdx
3. docs/introduction/intro/quickstart_3_backup.mdx

Please ensure you either:
- Redirect these references to an alternative page
- Create a replacement page at a new location
- Update all referring documents to remove these links